Rework communication docs, add Vault tutorial (#82)
infracloudio/botkube-docs: content/configuration/resource.md

---
title: "Resource"
date: 2019-01-08T17:58:08+05:30
draft: false
weight: 20
---

BotKube backend reads resource configurations from the [resource_config.yaml](#file-syntax) file placed in the directory specified by the **CONFIG_PATH** environment variable.

## File syntax

The resource configuration file contains:

- Resource list you want to watch,
- Namespaces you want to filter,
- The type of events you want to get notifications about,

> - `updateSettings` to monitor change in the specific resource field,

Reviewer: I know it's already existing content, but maybe we can reword this point to, e.g.:

```suggestion
- Settings to monitor updates for the specific resource fields,
```
